lib/public/views/Logs/Create/LogCreationModel.js

```diff
@@ -41,9 +41,10 @@ export class LogCreationModel extends Observable {
         this._creationTagsPickerModel.bubbleTo(this);
         this._creationTagsPickerModel.visualChange$.bubbleTo(this);
 
-        this._runNumbers = runNumbers.join(', ');
-        this._lhcFills = lhcFillNumbers.join(', ');
-        this._environments = environmentIds.join(', ');
+        this._dataDisplayDelimiter = ', ';
```

Comment:
Good idea, but I would rather use a constant in the top of the file

Reply:
refactored